src/Constants/constants.py

```diff
@@ -8,7 +8,11 @@
 DELETE_FREEZER = "delete freezer"
 DELETE_FRIDGE = "delete fridge"
 DELETE_PROFILE = "delete profile"
+EXPIRATION_DATE = "expiration date"
+EXPIRATION_WINDOW = "expiration window"
```

Comment:
These should both have underscores instead of spaces

Reply:
Done!